### PR TITLE
CI: use GitHub Actions for running tests on PR / Push events

### PR DESCRIPTION
**Pull request by XijiaoLi**
In this PR:
- Added basic Github Actions CI workflow config
  - os: ubuntu-18.04
  - resolver: lts-18.6

### CODE REVIEW

File: .github/workflows/tester.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      - 'test/**'
+      - 'unit-tests/**'
+
+jobs:
+  test:
+    name: Build and test sslang
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
```
Comment (j-hui) [inline .github/workflows/tester.yml:29]: Is there a reason this uses 18.04? Why not 20.04 (or whatever the latest LTS is)?

**Review by j-hui — APPROVED** [review]
Thanks for setting this up Xijiao! This is awesome (: 

Will approve now, but could you try to set this up for Ubuntu 20.04 before merging?

File: .github/workflows/tester.yml (hunk 1 above)
Comment (XijiaoLi) [reply .github/workflows/tester.yml:29]: No particular reason; I just chose it randomly xD. Should we change it to `ubuntu-latest` or `ubuntu-20.04` (explicitly fix the version)?

**Comment by j-hui** [comment]
Let's use ubuntu-latest.

**Pull request merged**